moduleframework/common.py

```diff
@@ -499,14 +501,23 @@ def getPackageList(self, profile=None):
         print_info("PCKGs to install inside module:", package_list)
         return package_list
 
-    def getModuleDependencies(self):
+    @property
+    def dependencylist(self):
```

Comment:
doesn't comply to [PEP8](https://www.python.org/dev/peps/pep-0008/#function-names):

    Function names should be lowercase, with words separated by underscores as necessary to improve readability.

Reply:
@jscotka I would like to test if dependency_list works properly. Any idea? Do we have a test for it already in `examples` directory?